packages/help-center/src/components/back-button.tsx

```diff
@@ -18,9 +22,32 @@ export const BackButton = () => {
 		}
 	}
 
+	useEffect( () => {
+		const nodeRef = backButtonRef;
+		const onBackButtonKeyDown = ( e: KeyboardEvent ) => {
+			if ( e.key === 'Enter' ) {
+				handleClick();
+			}
+		};
+
+		if ( nodeRef.current ) {
+			nodeRef.current?.addEventListener( 'keydown', onBackButtonKeyDown );
+			// Fixes accessibility for back button
+			nodeRef.current?.removeAttribute( 'aria-hidden' );
+		}
+		return () => {
+			nodeRef.current?.removeEventListener( 'keydown', onBackButtonKeyDown );
+		};
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [] );
```

Comment:
I don't think we need this. Pressing Enter fires onClick automatically.

Reply:
@alshakero - The onClick function doesn't trigger when I click `Enter` without this logic. I'm not sure if there's something I'm missing here.

Reply:
Checking..